mobile/src/main/res/xml/preferences.xml

```diff
@@ -133,6 +133,14 @@
             <!--android:key="@string/key_hideOnlineCards"-->
             <!--android:summary="@string/pref_hide_online_summary"-->
             <!--android:title="@string/pref_hide_online_title" />-->
+
+            <MultiSelectListPreference
+                android:dialogTitle="@string/pref_search_languages_title"
+                android:entries="@array/card_language"
+                android:entryValues="@array/card_language_values"
```

Comment:
Can you please add an `android:defaultValue` for English here? It's used elsewhere, you can search for `default_widget_buttons_array_entries` for an example.

Reply:
That's also what I used in my first draft, but changed it afterwards, see the next comment.